### PR TITLE
Feat: UI componnet for proxy account

### PR DESCRIPTION
**Pull request by egor0798**
## Changes:
- Added `ProxyType` enum and `ProxyTypeName` conts
- Added ProxyAccount UI component
- Storybook fix (but it's still not working)

*** 

image

### CODE REVIEW

**Comment by egor0798** [comment]
Task linked: [CU-869357eeu UI lib: Proxy account component](https://app.clickup.com/t/869357eeu)

**Review by tuul-wq — APPROVED** [review]

File: src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <Identicon className="inline-block" address={address} size={20} background={false} canCopy={canCopy} />
+      <div className="grid gap-y-0.5">
+        <BodyText className="text-text-secondary truncate">{name ? name : addressContent}</BodyText>
+        {name && <HelpText className="text-text-tertiary truncate">{addressContent}</HelpText>}
+        <div className="flex gap-x-1 items-center mt-0.5">
+          <span className="w-1 h-1 rounded-full bg-tab-text-accent" />
+          <HelpText className="text-tab-text-accent">{t(ProxyTypeName[proxyType])}</HelpText>
+        </div>
+      </div>
+      {actions && ActionButton}
```
Comment (Asmadek) [inline src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx:64]: You already have checking for `actions` do we really need double checking?

File: src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        ))}
+      </DropdownIconButton.Items>
+    </DropdownIconButton>
+  );
+
+  return (
+    <div className={cnTw('flex items-center gap-x-2', className)}>
+      <Identicon className="inline-block" address={address} size={20} background={false} canCopy={canCopy} />
+      <div className="grid gap-y-0.5">
+        <BodyText className="text-text-secondary truncate">{name ? name : addressContent}</BodyText>
```
Comment (Asmadek) [inline src/renderer/entities/proxy/ui/ProxyAccount/ProxyAccount.tsx:57]:
```suggestion
        <BodyText className="text-text-secondary truncate">{name ?? addressContent}</BodyText>
```

**Review by Asmadek — APPROVED** [review]
some comments

**Pull request merged**